frontend/public/components/operator-lifecycle-manager/install-plan.tsx

```diff
@@ -56,7 +56,7 @@ export const InstallPlanRow: React.SFC<InstallPlanRowProps> = (props) => {
   </ResourceRow>;
 };
 export const InstallPlansList = requireOperatorGroup((props: InstallPlansListProps) => {
-  const EmptyMsg = () => <MsgBox title="No Install Plans Found" detail="Install Plans are created automatically by subscriptions or manually using kubectl." />;
+  const EmptyMsg = () => <MsgBox title="No Install Plans Found" detail="Install Plans are created automatically by subscriptions or manually using the CLI." />;
```

Comment:
Compromise between `kubectl` and `oc` (since console works with upstream Kubernetes).